girder/utility/install.py

```diff
@@ -155,13 +155,26 @@ def runWebBuild(wd=None, dev=False, npm='npm', allPlugins=False, plugins=None, p
                             (' '.join(cmd), proc.returncode))
 
 
+def _runWatchCmd(*args):
+    try:
+        subprocess.Popen(args, cwd=constants.PACKAGE_DIR).wait()
+    except KeyboardInterrupt:
+        pass
+
+
 def install_web(opts=None):
     """
     Build and install Girder's web client. This runs `npm install` to execute
```

Comment:
Can you update this docstring?  `npm install` is no longer right, at least not on it's own, and maybe it's better to not mention it here.  Also, under certain conditions this does not build and install Girder's web client (when watching a plugin).  I'm not sure of a good docstring, maybe

   Build and install Girder's web client, or perform a watch on the web client or a specified plugin.

Otherwise I tested this out for main client and a plugin, it works for js, stylus and pug files.